everest

```diff
@@ -564,33 +564,43 @@ OCAML
   #
   # regtool get \HKLM\Software\Python\PythonCore\2.7\InstallPath\
   # regtool get \HKCU\Software\Python\PythonCore\2.7\InstallPath\
+  # CMW: The following are the 32-bit installations, I think.
   # regtool get \HKLM\Software\WOW6432Node\Python\PythonCore\2.7\InstallPath\
   # regtool get \HKCU\Software\WOW6432Node\Python\PythonCore\2.7\InstallPath\
   #
   #
   if is_windows; then
-    if ! [ -d "/cygdrive/c/Python27" ]; then
-      red "ERROR: C:\\Python27 not found -- install a *WINDOWS* Python 2.7? (y/n)"
+    if [ -z ${var+x} ]; then
```

Comment:
what does this test do?

Reply:
It checks whether "var" is assigned any value at all, i.e. power-users can overwrite it with their own custom directory. I forgot to turn `var` into `PY27DIR` though, good catch!